components/SlateRenderer/components/StoryBookmark.tsx

```diff
@@ -1,17 +1,18 @@
 import { Elements } from '@prezly/content-renderer-react-js';
 import type { StoryBookmarkNode } from '@prezly/slate-types';
-import { useEmbedStory } from '@prezly/theme-kit-nextjs';
+import { useCurrentStory } from '@prezly/theme-kit-nextjs';
 
 interface Props {
     node: StoryBookmarkNode;
 }
 
 export function StoryBookmark({ node }: Props) {
-    const embedStory = useEmbedStory(node.story.uuid);
+    const story = useCurrentStory();
+    const storyOEmbedInfo = story?.referenced_entities?.stories?.[node.story.uuid];
```

Comment:
Linter complains for a reason :)
Let's call it `referencedStory` for simplicity

Reply:
Also, was this API there before we implemented the embed stories API? 🤔